aptos-move/framework/aptos-framework/sources/atomic_bridge.move

```diff
@@ -1686,4 +1818,65 @@ module aptos_framework::atomic_bridge_counterparty {
 
         complete_bridge_transfer(bridge_transfer_id, b"not the secret");
     }
+
+    #[test(aptos_framework = @aptos_framework, third_party = @0xdead)]
+    /// Test eligible sponsor 
+    fun test_lock_for_eligible_sponsor(aptos_framework: &signer, third_party: &signer) acquires BridgeCounterpartyEvents {
```

Comment:
```suggestion
    fun test_lock_for_eligible_sponsorship(aptos_framework: &signer, third_party: &signer) acquires BridgeCounterpartyEvents {
```

This isn't a test of whether a specific sponsor is eligible, but whether a transfer is eligible for sponsorship.

Reply:
Agreed